### DISCUSSION TITLE
Feture Request: Upload images along the text when capture pages

### DISCUSSION THREAD

**Discussion by lumenpink** [coddingtonbear/obsidian-web#105] [Ideas]
Hi! Just wondering if it could be possible to send not only the text but images also.
I'm trying to keep all notes to myself and when I have no connection I can't see the images from captured sites because they are still kept pointing to the original site.

Thanks in advance.

**Comment by coddingtonbear** [comment]
For this kind of use case, I'd recommend using a tool like @Sergei-Korneev's https://github.com/Sergei-Korneev/obsidian-local-images-plus ; I believe there's a problem currently preventing its use with Obsidian Web (see https://github.com/Sergei-Korneev/obsidian-local-images-plus/issues/47) that prevents its use for the moment, but once that issue is resolved it seems like it'd solve your use case nicely.
